dags/glam_subdags/general.py

```diff
@@ -24,6 +24,7 @@ def repeated_subdag(
     additional_params=None,
     num_partitions=5,
     date_partition_parameter="submission_date",
+    docker_image="gcr.io/moz-fx-data-airflow-prod-88e0/bigquery-etl:latest",
```

Comment:
Adding the ability to pass `docker_image` to these subdags so they can pass it to `bigquery_etl_query` is an extra precaution that isn't completely necessary because there's no custom code in the dev docker image that touches the jobs in those subdags. The only important thing there is to change the dataset_id. 

I'm also doing this in case I do need to have custom code in those subdags, then I won't need to open another PR here to include the parameter. 

Those parameters will be removed after the tests are done. It does not impact glam prod (the default value is the prod image). The removal is being tracked [here](https://github.com/mozilla/glam/issues/2177)